container_pipeline/workers/base.py

```diff
@@ -46,20 +48,41 @@ def export_logs(self, logs, destination):
 
     def run(self):
         """Run worker"""
+        current_job_name = None
+        previous_job_name = None
+        current_trig_time = datetime.time
+        previous_trig_time = datetime.time
+
         while True:
-            job_obj = self.queue.get()
-            job = json.loads(job_obj.body)
-            debug_logs_file = os.path.join(
-                job['logs_dir'], settings.SERVICE_LOGFILE)
-            # Run dfh.clean() to clean log files if no error is encountered in
-            # post delivering build report mails to user
-            dfh = DynamicFileHandler(self.logger, debug_logs_file)
-            self.logger.info('Got job: {}'.format(job))
             try:
-                self.handle_job(job)
+                job_obj = None
+                previous_job_name = current_job_name
+                previous_trig_time = current_trig_time
```

Comment:
can we take these unnecessary operations out of try block ? We had problem earlier in the code where there were long `try` block code snippets. Lets make sure we not going there again.